[2022-03-20 20:52:39,972] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-15T00:00:00+00:00 [queued]>
[2022-03-20 20:52:40,006] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-15T00:00:00+00:00 [queued]>
[2022-03-20 20:52:40,009] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:52:40,009] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2022-03-20 20:52:40,009] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:52:40,037] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): prepare_insert> on 2022-03-15T00:00:00+00:00
[2022-03-20 20:52:40,052] {standard_task_runner.py:52} INFO - Started process 410 to run task
[2022-03-20 20:52:40,058] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_news', 'prepare_insert', '2022-03-15T00:00:00+00:00', '--job-id', '235', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/fetch_news.py', '--cfg-path', '/tmp/tmphe1g2lec', '--error-file', '/tmp/tmpt6nytbda']
[2022-03-20 20:52:40,058] {standard_task_runner.py:77} INFO - Job 235: Subtask prepare_insert
[2022-03-20 20:52:40,171] {logging_mixin.py:104} INFO - Running <TaskInstance: fetch_news.prepare_insert 2022-03-15T00:00:00+00:00 [running]> on host e39215f0be5b
[2022-03-20 20:52:40,326] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=fetch_news
AIRFLOW_CTX_TASK_ID=prepare_insert
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-15T00:00:00+00:00
[2022-03-20 20:52:40,328] {python.py:118} INFO - Done. Returned value was: None
[2022-03-20 20:52:40,399] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=fetch_news, task_id=prepare_insert, execution_date=20220315T000000, start_date=20220320T205239, end_date=20220320T205240
[2022-03-20 20:52:40,490] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-20 20:52:40,537] {local_task_job.py:146} INFO - Task exited with return code 0
